### DISCUSSION TITLE
Stellarium for Android without Google Play

### DISCUSSION THREAD

**Discussion by iio7** [Stellarium/stellarium#3252] [General]
I am using LineageOS on my phone and I do not have a Google account. I would like to be able to download an APK for Android in order to install it manually, or at least get it from F-Droid.

Could this be made possible?

**Comment by gzotti** [comment]
This github site is for the desktop version. We developers know nothing more than the Android version's existence. Your favourite search engine will help you better.

**Comment by kolAflash** [comment]
@iio7  
You can find some information about the mobile version here:  
https://en.wikipedia.org/wiki/Stellarium_(software)#Stellarium_Mobile  
https://www.stellarium-labs.com/  
They also provide a web version which runs in Firefox or Chrome.  
https://stellarium-web.org/

You can download the Android version via APK download websites like APK-Mirror. BUT BE CAREFUL NOT TO DOWNLOAD MALWARE!  
https://www.apkmirror.com/apk/stellarium-labs/stellarium-mobile-star-map/  
Alternatively you may use the Aurora-Store open source app to download it from PlayStore.  
https://f-droid.org/packages/com.aurora.store/

It looks like the Android version needs Google Play Services installed on the device. So you need at least a recent MicroG version of LineageOS (MicroG older than 1 year seems not to work).
https://en.wikipedia.org/wiki/MicroG  
https://download.lineage.microg.org/

&nbsp;

@gzotti  
As far as I know Stellarium Desktop is [GPL copyleft licensed](https://en.wikipedia.org/wiki/GNU_General_Public_License). So Stellarium Mobile must release the source for all their modifications. Do you know anything about this? I found source code up to Stellarium-Android version 1.10 here:  
https://web.archive.org/web/20140314141711/http://www.noctua-software.com/stellarium-mobile  
But I could not find the sources for the recent 1.11 version on the current website https://www.stellarium-labs.com/  

My hope is, that someone might compile a Stellarium-Android version which does not depend on Google Play Services. Anyone heared of that!?

**Reply by 10110111** [reply]
Stellarium Mobile doesn't share code with Stellarium desktop.

**Reply by gzotti** [reply]
There is a free mobile (Maemo, Android) version of Stellarium that was derived from (AFAIK) and shares code with the 0.11/0.12 series. I don't know the state of updates. The current "mobile" version is independent from this repo. Version numbers like "1.10 for Android" are not related to our version 1.1 here, and I cannot give any information what these Android numbers mean. 
There are occasional discussions like #2226, or even a stale PR #194 which may have been targeted at large-screen Android tablets, but still did not provide a smartphone optimized interface. There are forks which usually lead nowhere. But there is no active development of an official Android port in this repository.

**Comment by tacitvenom** [comment]
@iio7 Did you manage to find a solution? I am in the same boat.

**Reply by khlsvr** [reply]
I have the OS that comes with the honor phones, but I have not logged into Google Play Store. I am using Aurora Store and F-Droid for my apps. 

The first app I have come across so far that I can't use without this combination is Stellarium. For some reason after installing the app from the Aurora Store (or installing .apkm downloaded from apkmirror) and then launching it, it will pop up the google play store app with "sign in" button. If I close this window, the Stellarium window also closes in the background. So even with this combination I haven't found a way to use the Stellarium Mobile app.

**Reply by gzotti** [reply]
You are still barking up the wrong tree here. See above for smartphone-related solutions or websites. This site discusses Stellarium for desktop systems.

**Reply by khlsvr** [reply]
I read those tips and commented upon them. I know this is a wrong place too, but I thought it's the best we've got, since it seems like there is no separate project for the mobile app at https://github.com/Stellarium and other people were commenting here aswell. Is there a right tree perhaps somewhere else on github or elsewhere? Just wanted to mention that the mobile solutions pointed out by @kolAflash are not helping even if you have google service framework installed, but you don't want to sign into google play store.

**Reply by alex-w** [reply]
https://stellarium-labs.com/stellarium-mobile-plus/

**Reply by khlsvr** [reply]
Thanks. They only have an e-mail address, which makes it harder for a community to engage in conversation. I see it's not wise or welcome to talk about an unrelated project here and I can see that the reddit's r/stellarium subreddit would be more appropriate place for something like this so maybe some of us should start a thread in there to move this discussion there.
